### PR TITLE
sci-mathematics/giac: bug-fix for bugzilla bug 598008

### PR DESCRIPTION
**Pull request by timeraider4u**
Package-Manager: portage-2.3.0

### CODE REVIEW

File: sci-mathematics/giac/giac-1.1.0.ebuild [hunk 1]
```diff
@@ -50,7 +50,7 @@ src_configure(){
 
 src_install() {
 	emake install DESTDIR="${D}"
-	mv "${D}"/usr/bin/{aide,giac-help} || die
+	cp "${S}/src/aide" "${D}"/usr/bin/ || die
```
Comment (marbre) [inline sci-mathematics/giac/giac-1.1.0.ebuild]: Why not `dobin`?

Comment (timeraider4u) [inline sci-mathematics/giac/giac-1.1.0.ebuild]: Thx, you are right. Corrected with the current rebased commit.

**Comment by kiwifb** [comment]
Any reason you are sticking to 1.1.0 rather than go say 1.2.2-103? That's the latest version I am now carrying in the sage-on-gentoo overlay.

**Comment by timeraider4u** [comment]
well, I am not a maintainer of this ebuild, just stumbled over this issue and provided a fix ;-).

**Comment by timeraider4u** [comment]
thx for merging and closing the corresponding Bugzilla bug.

**Comment by marbre** [comment]
@timeraider4u Thanks for the fix. If you like, feel free to bump to 1.2.2-103 as @kiwifb suggested ;)

**Comment by mmokrejs** [comment]
Marius Brehler wrote:
> *@marbre* requested changes on this pull request.
> In sci-mathematics/giac/giac-1.1.0.ebuild <https://github.com/gentoo-science/sci/pull/695#pullrequestreview-9819122>:
>
>> @@ -50,7 +50,7 @@ src_configure(){
>
>  src_install() {
>  	emake install DESTDIR="${D}"
> -	mv "${D}"/usr/bin/{aide,giac-help} || die
> +	cp "${S}/src/aide" "${D}"/usr/bin/ || die
>
> Why not |dobin|?

Replacing ${D} with ${ED} on the "mv" commandline would seem a good idea so that the ebuild works on Gentoo::Prefix installations as well. The dobin on the next line should prepend the ${EPREFIX} in front of "${D}" automatically/internally.

**Pull request merged**